scripts/#4 Users with more than one order.sql

```diff
@@ -0,0 +1,5 @@
+SELECT users.firstname, users.lastname, COUNT(orders.orderid) AS count
+FROM users LEFT JOIN orders
+ON users.userid = orders.userid
+GROUP BY users.lastname

```

Comment:
Is this script work and return the result for you? I think you cannot group by only last name when you want to display both first and last names.